### DISCUSSION TITLE
Do we have any plans to support multiple languages for the document?

### DISCUSSION THREAD

**Discussion by LinuxSuRen** [argoproj/argo-cd#7886] [General]
As I mentioned in the title. I adopt Argo CD as the GitOps framework in my [infra](https://github.com/kubesphere-sigs/ks-infra/). And [we](https://github.com/kubesphere/ks-devops/) are going to make Argo CD as the default CD solution.

Yes, I have read a lot of Argo CD documents. I'm wondering if it's possible to provide the Chinese version for more potential Chinese users (and developers).

We already recorded [some video tutorials](https://www.bilibili.com/video/BV17F411h7Zh) (in Chinese) of Argo CD.

**Comment by alexmt** [comment]
Hello @LinuxSuRen , Argo CD documentation is managed using [readthedocs](https://readthedocs.org/). Apparently, it supports localization : https://docs.readthedocs.io/en/stable/localization.html . So it is totally possible to add Chinese version.

We are planning to refresh the documentation structure https://github.com/argoproj/argo-cd/issues/5635 (  I really hope in during the next release ). I would suggest starting working on translation after the documentation structure is updated.

**Reply by LinuxSuRen** [reply]
Thanks for your response. I'm looking forward it.

**Comment by artemus717** [comment]
Hi @alexmt  Did the translation to chinese finished ? I want to do something for help , I am working at an International company .
